README.md

```diff
@@ -141,22 +147,28 @@ Request Body: {"declaration":{"attributes":{"version":"1.0"}},"elements":[{"type
 - `includeUnparsed` **{Boolean}** Toggles co-body returnRawBody option; if set to true, for form encoded and JSON requests the raw, unparsed request body will be attached to `ctx.request.body` using a `Symbol` ([see details](#a-note-about-unparsed-request-bodies)), default `false`
 - `formidable` **{Object}** Options to pass to the formidable multipart parser
 - `onError` **{Function}** Custom error handle, if throw an error, you can customize the response - onError(error, context), default will throw
-- `strict` **{Boolean}** ***DEPRECATED*** If enabled, don't parse GET, HEAD, DELETE requests, default `true`
+- `strict` **{Boolean}** **_DEPRECATED_** If enabled, don't parse GET, HEAD, DELETE requests, default `true`
```

Comment:
I think we can delete documentation of this option

Reply:
done 👍